.github/workflows/ci.yml

```diff
@@ -1,25 +1,27 @@
 name: Build and Test
 
-on: [push, pull_request] 
+on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    
-    - name: Setup .NET Core
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: 3.1.101
-    
-    - name: Install dependencies
-      run: dotnet restore
-    
-    - name: Build
-      run: dotnet build --configuration Release --no-restore
-    
-    - name: Test
-      run: dotnet test --no-restore --verbosity normal
+      - uses: actions/checkout@v2
+
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.101
+
+      - name: Install dependencies
+        run: dotnet restore
+
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+
+      - name: Test
+        run: dotnet test --no-restore --verbosity normal
+        env:
+          CONNECTION: ${{ secrets.CONNECTION }}
+          GITHUB: true
```

Comment:
I believe the github flag was only needed for the excel testing.  I'm not seeing it referenced in the code you're committing.

Reply:
nevermind

Reply:
It's used in the configuration to point to the correct appsettings